modules/nf-core/clustalo/align/tests/main.nf.test

```diff
@@ -0,0 +1,70 @@
+nextflow_process {
+
+    name "Test Process CLUSTALO_ALIGN"
+    script "../main.nf"
+    process "CLUSTALO_ALIGN"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "clustalo"
+    tag "clustalo/align"
+
+    test("sarscov2 - contigs-fasta") {
+
+        when {
+            process {
+                """
+                input[0] = [ [ id:'test' ], // meta map
+                             file(params.test_data['sarscov2']['illumina']['contigs_fasta'], checkIfExists: true)
+                           ]
+                input[1] = [[:],[]]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out.alignment).match("alignment")},
+                { assert snapshot(process.out.versions).match("versions") }
+            )
+        }
+
+    }
+
+    test("sarscov2 - contigs-fasta - guide_tree") {
+
+        setup {
+
+            run("CLUSTALO_GUIDETREE") {
+                script "../../guidetree/main.nf"
+                process {
+                    """
+                    input[0] = [ [ id:'test' ], // meta map
+                                 file(params.test_data['sarscov2']['illumina']['contigs_fasta'], checkIfExists: true)
+                               ]
+                    """
+                }
+            }
+        }
+
+        when {
+            process {
+                """
+                input[0] = [ [ id:'test' ], // meta map
+                             file(params.test_data['sarscov2']['illumina']['contigs_fasta'], checkIfExists: true)
+                           ]
+                input[1] = CLUSTALO_GUIDETREE.out.tree.collect{ meta, tree -> tree }.map{ tree -> [[ id: 'test_summary'], tree]}
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out.alignment).match("alignment")},
+                { assert snapshot(process.out.versions).match("versions") }
```

Comment:
```suggestion
                { assert snapshot(process.out.versions).match("with_guide_tree_versions") }
```

Reply:
Fixed in [725039e](https://github.com/nf-core/modules/pull/4426/commits/725039e834919815e2dd68d76a5cddda2f99af5c)